crypto/paillier/paillier.go

```diff
@@ -65,7 +65,7 @@ func init() {
 }
 
 // len is the length of the modulus (each prime = len / 2)
-func GenerateKeyPair(modulusBitLen int, timeout time.Duration, optionalConcurrency ...int) (privateKey *PrivateKey, publicKey *PublicKey, err error) {
```

Comment:
This function is public so this is a potential breaking change. Shouldn't we expose ` GenerateKeyPairWithContext` for full backward compatibility?